File: uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.cs
public class Solution 
{
    public string[] UncommonFromSentences(string A, string B) 
    {
        string ab = string.Concat(A, " ", B);
        List<string> res = new List<string>();
        Dictionary<string, int> dict = new Dictionary<string, int>();
        foreach(var word in ab.Split(" "))
        {
            if(dict.ContainsKey(word))
            {
                dict[word]++;
                res.Remove(word);
            }
            else
            {
                dict.Add(word,1);
                res.Add(word);
            }    
        }
        return res.ToArray();
    }
}

File: uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.js
/**
 * @param {string} s1
 * @param {string} s2
 * @return {string[]}
 */
var uncommonFromSentences = function(s1, s2) {
    const arr = [], res = {};
    
    s1.concat(" ", s2).split(" ").forEach(ele => {
        res[ele] = res[ele] ? res[ele] + 1 : 1;
    });
    
    for (const [key, value] of Object.entries(res)) {
        if(value == 1)
        {
            arr.push(key);
        }
    }
    
    return arr;
};